src/app/dataset-block/metadata-block/components/event-details/components/common/interval-property/interval-property.component.ts

```diff
@@ -0,0 +1,16 @@
+import { BasePropertyComponent } from "src/app/dataset-block/metadata-block/components/event-details/components/common/base-property/base-property.component";
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import {
+    OffsetInterval,
+    BlockInterval,
+} from "src/app/api/kamu.graphql.interface";
+
+@Component({
+    selector: "app-interval-property",
+    templateUrl: "./interval-property.component.html",
+    styleUrls: ["./interval-property.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class IntervalPropertyComponent extends BasePropertyComponent {
+    @Input() public data: OffsetInterval | BlockInterval;
+}
```

Comment:
Unfortunately, such a trick won't work.
For block intervals we would like to have block links.
For data offset intervals we would finally like to have a redirection to SQL query for this interval.